### DISCUSSION TITLE
Join the Lightning AI Ambassador Program - Lightning League ⚡

### DISCUSSION THREAD

**Discussion by naisofly** [Lightning-AI/pytorch-lightning#14939] [Show off your work]
Are you building Lightning Apps?⚡️

AND/OR

Enjoy sharing your machine learning knowledge with others?🧠💕

Apply to join our ambassador program, the Lightning League!

Learn more and apply now:
👉👉👉[https://bit.ly/3qEVJVo](https://t.co/FH9fRxIllY)

![Join the Lightning League ambassador program!](https://user-images.githubusercontent.com/5947670/193020478-4f070b0c-279f-4e64-aa85-87e262403fbf.gif)

**Comment by Keiku** [comment]
I received an e-mail on 09/16/2022 and replied with a Google form, but more than two weeks have passed and I still haven't received a reply. what should i do? Does this mean that it has been rejected?

**Comment by naisofly** [comment]
Hey @Keiku, it means your application is still under consideration :) 

We'll be reaching out to all the applicants within the week to let them know if they're accepted into the program & the next steps. Appreciate your patience & understanding!
